Add deprecation warning (#3490)
kyma-project/control-plane: README.md

## :warning: **Archived**

> This repository has been deprecated and archived on Feb 11th, 2025. There are replacements available, see 

Reviewer: ```suggestion
This repository has been deprecated and archived on Feb 11th, 2025. For available replacements, see the following:
```
